code/DDSCodeTester.cpp

```diff
@@ -416,6 +416,32 @@ void dds_domain_examples()
         //!--
     }
 
+    {
+        //DDS_CHANGE_DOMAINPARTICIPANTEXTENDEDQOS
+        // Create a DomainParticipant with DomainParticipantExtendedQos from profile
+        DomainParticipantExtendedQos profile_extended_qos;
+        DomainParticipantFactory::get_instance()->get_participant_extended_qos_from_profile("xml_profile.xml",
```

Comment:
Here, it's the name of the profile 
```suggestion
        DomainParticipantFactory::get_instance()->get_participant_extended_qos_from_profile("participant_profile",
```